app/src/main/java/app/rive/runtime/example/MetricsActivity.kt

```diff
@@ -11,18 +11,32 @@ class MetricsActivity : AppCompatActivity() {
     private val containerView by lazy(LazyThreadSafetyMode.NONE) {
         findViewById<LinearLayout>(R.id.container)
     }
+//
```

Comment:
Cleanup! Any reason to leave it hanging around? Otherwise, we can just delete it